[BUGFIX] instantiation of ExpectationSuite always adds ge version metadata to prevent datadocs from crashing

Aylr opened the pull request:
Changes proposed in this pull request:
- instantiation of ExpectationSuite always adds ge version metadata to prevent datadocs from crashing. Fixes #1637 

Previous Design Review notes:
- n/a

Aylr committed:
* instantiation of ExpectationSuite always adds ge version metadata (if not present) to prevent datadocs from crashing

roblim reviewed (changes requested):
LGTM! Looks like you just need to do black/isort and fix one test that's failing now - `def test_meta_version_warning()` - can prob nix it I think.

Aylr commented:
Thanks! I fixed the linting and the broken test, which actually did it's job and alerted me that I almost introduced a new bug where I would have overwritten the version to always be the latest. I updated that test to capture the intended behavior.

Aylr committed:
Merge branch 'develop' into bugfix/missing-metadata-crashes-datadocs

Aylr committed:
Merge branch 'develop' into bugfix/missing-metadata-crashes-datadocs

roblim reviewed (approved):
LGTM!
